scripts/apps/archive/directives/MediaPreview.ts

```diff
@@ -90,6 +91,27 @@ export function MediaPreview(api, $rootScope, desks, superdesk, content, storage
                 return _.map(scope.item.company_codes, 'qcode').join(', ');
             };
 
+            // Check if there is any association for fieldId
+            scope.associationExists = function(associations, fieldId) {
+                return _.size(scope.getAssociatedItems(associations, fieldId));
+            };
+
+            // Return all associations for fieldId
+            scope.getAssociatedItems = function(associations, fieldId) {
+                var result = _.filter(associations, (association, key) => key.indexOf(fieldId) !== -1);
+
+                return result;
+            };
+
+            // Check if the rendition is image or not
+            scope.isImage = (rendition) => isImage(rendition);
```

Comment:
Prefer code over comments. You could write it like this and it would be clear enough without any comments.

```
// utils file
export const renditions = {
    isImage: ...,
    isVideo: ...,
    isAudio: ...,
};
// this file
scope.renditions = renditions;
```